src/tools/schema2kotlin.ts

```diff
@@ -256,7 +306,16 @@ ${this.opts.wasm ? `
 }
 
 class ${name}_Spec() : ${this.getType('EntitySpec')}<${name}> {
+${this.opts.wasm ? '' : `\
 
```

Comment:
I think EntitySpec needs a method to retrieve this schema as well like

interface EntitySpec { 
   fun schema() : Schema
}

Because otherwise any runtime code that isn't code gen'ed that needs to look up the schema without knowing the hash, but having only the HandleHolder, won't know which Companion object to reference.

Reply:
Implemented.